app/services/nomis/client.rb

```diff
@@ -80,17 +81,27 @@ def request(method, route, params, idempotent:)
     # rubocop:enable Metrics/MethodLength
     # rubocop:enable Metrics/AbcSize
 
+    def http_method_expects(method)
+      if method == :get
+        [200]
+      else
+        # TODO: Change 400 to 422 when api is updated.
+        # Currently it returns 400 for validation errors.
+        [200, 400, 409]
+      end
+    end
+
     # Returns excon options which put params in either the query string or body.
     def params_options(method, params)
       return {} if params.empty?
 
-      if %i[get delete].include?(method)
+      if method == :post
```

Comment:
When we start using those we can reconsider this code, no point worrying about it now